lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,605 @@
+    type: str
+    aliases: [ name ]
+    version_added: "2.10"
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed
+      - Can't be changed with helm 2
+    default: "default"
+    required: false
+    type: str
+    aliases: [ namespace ]
+    version_added: "2.10"
+  release_state:
+    choices: ['present', 'absent']
+    description:
+      - Desirated state of release
+    required: false
+    default: present
+    aliases: [ state ]
+    version_added: "2.10"
+    type: str
+  release_values:
+    description:
+        - Value to pass to chart
+    required: false
+    default: {}
+    aliases: [ values ]
+    version_added: "2.10"
+    type: dict
+  tiller_host:
+    description:
+      - Address of Tiller
+      - Ignored when is helm 3
+    type: str
+    version_added: "2.10"
+  tiller_namespace:
+    description:
+      - Namespace of Tiller
+      - Ignored when is helm 3
+    default: "kube-system"
+    type: str
+    version_added: "2.10"
+  update_repo_cache:
+    description:
+      - Run `helm repo update` before the operation. Can be run as part of the package installation or as a separate step.
+    default: "false"
+    type: bool
+    version_added: "2.10"
+
+#Helm options
+  disable_hook:
+    description:
+      - Helm option to disable hook on install/upgrade/delete
+    default: False
+    type: bool
+    version_added: "2.10"
+  force:
+    description:
+      - Helm option to force reinstall, ignore on new install
+    default: False
+    type: bool
+    version_added: "2.10"
+  kube_context:
+    description:
+      - Helm option to specify which kubeconfig context to use
+    type: str
+    version_added: "2.10"
+  kubeconfig_path:
+    description:
+      - Helm option to specify kubeconfig path to use
+    type: path
+    version_added: "2.10"
+    aliases: [ kubeconfig ]
+  purge:
+    description:
+      - Remove the release from the store and make its name free for later use
+    default: True
+    type: bool
+    version_added: "2.10"
+  wait:
+    description:
+      - Wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment are in a ready state before marking the release as successful
+    default: False
+    type: bool
+    version_added: "2.10"
+  wait_timeout:
+    description:
+      - Timeout when wait option is enabled (helm2 is a number of seconds, helm3 is a duration
+    type: str
+    version_added: "2.10"
+'''
+
+EXAMPLES = '''
+# Deploy grafana with params version
+- helm_cli:
+    name: test
+    chart_ref: stable/grafana
+    chart_version: 3.3.8
+    tiller_namespace: helm
+    values:
+      replicas: 2
+
+# Load Value from template
+- helm_cli:
+    name: test
+    chart_ref: stable/grafana
+    tiller_namespace: helm
+    values: "{{ lookup('template', 'somefile.yaml') | from_yaml }}"
+
+# Remove test release
+- helm_cli:
+    name: test
+    state: absent
+    tiller_namespace: helm
+
+# With Helm3
```

Comment:
Since helm3 doesn't create namespaces, maybe add in a task that creates the `monitoring` namespace using the `k8s` module?

Reply:
Added too !